packages/botonic-cli/templates/custom-webchat/tests/app.test.js

```diff
@@ -1,69 +1,14 @@
-import { BotonicInputTester, BotonicOutputTester } from '@botonic/react'
+import { routes, plugins, locales, config } from '../src/'
+import {
+  BotonicInputTester,
+  BotonicOutputTester,
+  NodeApp,
+} from '@botonic/react'
 
-import App from '../src/app'
+const app = new NodeApp({ routes, locales, plugins, ...config })
 
-let i = new BotonicInputTester(App)
```

Comment:
can you comment on the PR description why these tests were removed?

Reply:
@dpinol, it's just that these tests were taken from the template `childs` and the code has no correspondences with the code itself of this template